components/TransLine.tsx

```diff
@@ -42,11 +52,15 @@ export const TransLine: React.FC<TransLineProps> = ({
   if (links && links.length > 0) {
     for (const link of links) {
       const href = resolveUrl(link, userArrivedFromUioMobile)
-      // Disabling some linting rules for this line. The anchor <a> element will
-      // be interpolated by <Trans>.
+      // Disabling some linting rules for the <a> lines. The anchor <a> element will
+      // be interpolated by <Trans> to have content.
       /* eslint-disable jsx-a11y/anchor-has-content */
       /* eslint-disable react/self-closing-comp */
-      linkComponents.push(<a href={href} key={link}></a>)
+      if (internalLink(link)) {
+        linkComponents.push(<a href={href} key={link}></a>)
+      } else {
+        linkComponents.push(<a target="_blank" rel="noopener noreferrer" href={href} key={link}></a>)
+      }
```

Comment:
I think what you wrote is better because it's more readable, but TIL I learned you could also do:

`linkComponents.push(<a href={href} key={link} {...(internalLink(link) ? {target:'_blank', rel:'noopener noreferrer'} : {})}></a>)`

Reply:
that is some perl-worthy density!